db/db_impl/db_impl.cc

```diff
@@ -3127,6 +3127,12 @@ bool CfdListContains(const CfdList& list, ColumnFamilyData* cfd) {
 }  //  namespace
 
 void DBImpl::ReleaseSnapshot(const Snapshot* s) {
+  if (s == nullptr) {
+    // DBImpl::GetSnapshot() can return nullptr when snapshot
+    // not supported by specifying the condition: memtable and
```

Comment:
Should the `memtable and` be removed or rephrased? I guess you are trying to indicate `inplace_update_support` is a memtable option?